docs/components/console/manage-clusters/settings.md

```diff
@@ -1,13 +1,52 @@
 ---
 id: settings
 title: Settings
-description: "Set up resource-based authorizations to authorize access and assign permissions to process and decision definitions."
+description: "Manage your cluster settings or permanently delete the cluster."
 ---
 
-Set up resource-based authorizations to authorize access and assign permissions to process and decision definitions in the **Settings** tab. You can enable resource-based authorizations on a per-cluster basis. It's possible to disable the feature at any time.
+Manage your cluster settings or permanently delete the cluster.
 
 ![Cluster settings](./img/cluster-settings.png)
 
+## Resource-based authorizations
+
+You can enable resource-based authorization on a per-cluster basis to control the level of access users have to system resources.
+
+- Enable this setting to use resource-based authorization in the cluster. You must define resource access for each user as users no longer have access to all process and decision definitions.
+- Disable this setting if you do not want to use resource-based authorization in the cluster. You can still configure resource authorizations, but they are only applied to cluster users when you enable this setting.
```

Comment:
Wait... really? Is this clear in the product experience?

Reply:
I assume you are referring to "You cna still configure resource..."? If so, I took this directly from the product, on the Settings page itself: 
<img width="902" alt="Screenshot 2024-07-24 at 09 23 19" src="https://github.com/user-attachments/assets/561e1748-efb3-444d-9c8a-3c360979bd83">

Reply:
Ok cool, happy to see this called out.